plugins/destination/mssql/client/client.go

```diff
@@ -5,15 +5,17 @@ import (
 	"database/sql"
 	"fmt"
 
+	"github.com/cloudquery/cloudquery/plugins/destination/mssql/queries"
 	"github.com/cloudquery/plugin-sdk/plugins/destination"
 	"github.com/cloudquery/plugin-sdk/specs"
 	mssql "github.com/microsoft/go-mssqldb"
 	"github.com/rs/zerolog"
 )
 
 type Client struct {
-	db         *sql.DB
-	schemaName string
+	db           *sql.DB
+	schemaName   string
+	databaseName string
```

Comment:
(applies to all such occasions).
Additionally, we don't need to specify the database name at all, as the conn string has to handle this itself.

For manual testing of queries you can use option to select db & schema.